source/admin_manual/doveadm_http_api.rst

```diff
@@ -3381,6 +3530,43 @@ parameters::
 +------------+--------+------------------------+---------------------------------+
 
 
+doveadm metacache update userdb
+===============================
+
+Update alternative usernames (user_* fields) in doveadm metacache list output.
```

Comment:
Doesn't this need escaping? `\*`

Reply:
Compiles and displays fine, so there must be some kind of limiting logic for scope of "*"...

And escaping the * gives: `/doveadm_http_api.rst:3536:Unknown target name: "user".`

So probably best to just leave it alone?